app/src/main/java/at/ac/fhcampuswien/budget_fox/widgets/SimpleEventIcon.kt

```diff
@@ -0,0 +1,27 @@
+package at.ac.fhcampuswien.budget_fox.widgets
+
+import androidx.compose.material3.Icon
+import androidx.compose.material3.IconButton
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.vector.ImageVector
+
+@Composable
+fun SimpleEventIcon(
+    icon: ImageVector,
+    modifier: Modifier = Modifier,
+    colour: Color = Color.Unspecified,
+    event: () -> Unit
+) {
+    IconButton(
+        onClick = event,
+        modifier = modifier
+    ) {
+        Icon(
+            imageVector = icon,
+            contentDescription = "Localized description",
```

Comment:
Not a fan of Localized description -> not a barrier free solution for example screen readers (important for WPF2)